otoroshi/app/api.scala

```diff
@@ -443,7 +443,7 @@ class ProgrammaticOtoroshiComponents(_serverConfig: play.core.server.ServerConfi
   override lazy val httpErrorHandler: HttpErrorHandler     = wire[ErrorHandler]
   override lazy val serverConfig                           = _serverConfig
 
-  lazy val handlerRef = new AtomicReference[HttpRequestHandler](httpRequestHandler)
+  lazy val handlerRef = new AtomicReference[HttpRequestHandler]()
```

Comment:
I'm not sure at all for this one, but without this running tests throws a Stackoverflow error at startup. I don't understand why it doesn't do so for "normal run"